src/main/java/iudx/resource/server/apiserver/ApiServerVerticle.java

```diff
@@ -530,8 +530,15 @@ private Future<Void> getConsumerAuditDetail(RoutingContext routingContext) {
           handler -> {
             if (handler.succeeded()) {
               LOGGER.debug("Table Reading Done.");
-              handleSuccessResponse(
-                  response, ResponseType.Ok.getCode(), handler.result().toString());
+              JsonObject jsonObject = (JsonObject) handler.result();
+              String s = jsonObject.getString("type");
+              if (s.equalsIgnoreCase("204")) {
```

Comment:
Use Constants file for constants like 204,200 etc.

Reply:
Solved